docs/data-sources/loadbalancer.md

```diff
@@ -34,5 +34,5 @@ The following arguments are supported:
 
 ## Attributes Reference
 
-See the [Load Balancer Resource](/docs/providers/do/r/loadbalancer.html) for details on the
+See the [Load Balancer Resource](/docs/resources/loadbalancer) for details on the
```

Comment:
```suggestion
See the [Load Balancer Resource](/providers/digitalocean/digitalocean/latest/docs/resources/loadbalancer) for details on the
```

Looks like the layout in the Terraform registry requires something slightly different.  As written, this would produce a link to: https://registry.terraform.io/docs/resources/loadbalancer

Reply:
Thanks for the feedback, I should have done a better job testing these changes.